--- gnome/src/Makefile.1	2009-09-03 21:09:13.000000000 +0700
+++ gnome/src/Makefile	2009-09-03 21:09:35.000000000 +0700
@@ -162,4 +162,4 @@
 
 ###
 install:
-	$(INSTALL_DATA) ../libgnome-2.0-ruby.pc $(libdir)/pkgconfig
+	$(INSTALL_DATA) ../libgnome-2.0-ruby.pc -D $(libdir)/pkgconfig/libgnome-2.0-ruby.pc
--- gnome/src/depend.1	2009-09-03 21:09:15.000000000 +0700
+++ gnome/src/depend	2009-09-03 21:09:45.000000000 +0700
@@ -1,2 +1,2 @@
 install:
-	$(INSTALL_DATA) ../libgnome-2.0-ruby.pc $(libdir)/pkgconfig
+	$(INSTALL_DATA) ../libgnome-2.0-ruby.pc -D $(libdir)/pkgconfig/libgnome-2.0-ruby.pc
