--- gnomevfs/src/Makefile.1	2009-09-03 21:23:37.000000000 +0700
+++ gnomevfs/src/Makefile	2009-09-03 21:23:59.000000000 +0700
@@ -162,4 +162,4 @@
 
 ###
 install:
-	$(INSTALL_DATA) ../gnome-vfs-2.0-ruby.pc $(libdir)/pkgconfig
+	$(INSTALL_DATA) ../gnome-vfs-2.0-ruby.pc -D $(libdir)/pkgconfig/gnome-vfs-2.0-ruby.pc
--- gnomevfs/src/depend.1	2009-09-03 21:23:40.000000000 +0700
+++ gnomevfs/src/depend	2009-09-03 21:24:13.000000000 +0700
@@ -1,2 +1,2 @@
 install:
-	$(INSTALL_DATA) ../gnome-vfs-2.0-ruby.pc $(libdir)/pkgconfig
+	$(INSTALL_DATA) ../gnome-vfs-2.0-ruby.pc -D $(libdir)/pkgconfig/gnome-vfs-2.0-ruby.pc
