--- gtkglext/src/Makefile.1	2009-09-03 23:57:48.000000000 +0700
+++ gtkglext/src/Makefile	2009-09-03 23:58:09.000000000 +0700
@@ -162,4 +162,4 @@
 
 ###
 install:
-	$(INSTALL_DATA) ../gtkglext-1.0-ruby.pc $(libdir)/pkgconfig
+	$(INSTALL_DATA) ../gtkglext-1.0-ruby.pc -D $(libdir)/pkgconfig/gtkglext-1.0-ruby.pc
--- gtkglext/src/depend.1	2009-09-03 23:57:52.000000000 +0700
+++ gtkglext/src/depend	2009-09-03 23:58:18.000000000 +0700
@@ -1,2 +1,2 @@
 install:
-	$(INSTALL_DATA) ../gtkglext-1.0-ruby.pc $(libdir)/pkgconfig
+	$(INSTALL_DATA) ../gtkglext-1.0-ruby.pc -D $(libdir)/pkgconfig/gtkglext-1.0-ruby.pc
