--- gtkmozembed/src/Makefile.1	2009-09-03 21:28:27.000000000 +0700
+++ gtkmozembed/src/Makefile	2009-09-03 21:29:00.000000000 +0700
@@ -162,4 +162,4 @@
 
 ###
 install:
-	$(INSTALL_DATA) ../mozilla-gtkmozembed-ruby.pc $(libdir)/pkgconfig
+	$(INSTALL_DATA) ../mozilla-gtkmozembed-ruby.pc -D $(libdir)/pkgconfig/mozilla-gtkmozembed-ruby.pc
--- gtkmozembed/src/depend.1	2009-09-03 21:28:31.000000000 +0700
+++ gtkmozembed/src/depend	2009-09-03 21:29:15.000000000 +0700
@@ -1,2 +1,2 @@
 install:
-	$(INSTALL_DATA) ../mozilla-gtkmozembed-ruby.pc $(libdir)/pkgconfig
+	$(INSTALL_DATA) ../mozilla-gtkmozembed-ruby.pc -D $(libdir)/pkgconfig/mozilla-gtkmozembed-ruby.pc
