--- gtk/src/Makefile.ori	2009-09-03 20:43:58.000000000 +0700
+++ gtk/src/Makefile	2009-09-03 20:44:24.000000000 +0700
@@ -175,4 +175,4 @@
 	$(INSTALL_DATA) $(srcdir)/rbgtkmacros.h $(RUBYARCHDIR)
 
 install:
-	$(INSTALL_DATA) ../gtk+-2.0-ruby.pc $(libdir)/pkgconfig
+	$(INSTALL_DATA) ../gtk+-2.0-ruby.pc -D $(libdir)/pkgconfig/gtk+-2.0-ruby.pc
--- gtk/src/depend.ori	2009-09-03 20:44:05.000000000 +0700
+++ gtk/src/depend	2009-09-03 20:44:38.000000000 +0700
@@ -6,4 +6,4 @@
 	$(INSTALL_DATA) $(srcdir)/rbgtkmacros.h $(RUBYARCHDIR)
 
 install:
-	$(INSTALL_DATA) ../gtk+-2.0-ruby.pc $(libdir)/pkgconfig
+	$(INSTALL_DATA) ../gtk+-2.0-ruby.pc -D $(libdir)/pkgconfig/gtk+-2.0-ruby.pc
