--- bonobo/src/Makefile	2009-09-03 19:37:12.000000000 +0700
+++ bonobo/src/Makefile	2009-09-03 19:38:44.000000000 +0700
@@ -162,7 +162,7 @@
 
 ###
 install:
-	$(INSTALL_DATA) ../libbonobo-2.0-ruby.pc $(libdir)/pkgconfig
+	$(INSTALL_DATA) ../libbonobo-2.0-ruby.pc -D $(libdir)/pkgconfig/libbonobo-2.0-ruby.pc
 rbbonobo-main.o: rbbonobo-main.c rbbonobo.h 
 rbbonobo.o: rbbonobo.c rbbonobo.h 
 rbbonobo-i18n.o: rbbonobo-i18n.c rbbonobo.h 
--- bonobo/src/depend	2009-07-26 13:16:56.000000000 +0700
+++ bonobo/src/depend	2009-09-03 19:38:58.000000000 +0700
@@ -1,2 +1,2 @@
 install:
-	$(INSTALL_DATA) ../libbonobo-2.0-ruby.pc $(libdir)/pkgconfig
+	$(INSTALL_DATA) ../libbonobo-2.0-ruby.pc -D $(libdir)/pkgconfig/libbonobo-2.0-ruby.pc
