--- glib/src/Makefile.ori	2009-09-03 19:00:03.000000000 +0700
+++ glib/src/Makefile	2009-09-03 19:00:22.000000000 +0700
@@ -178,4 +178,4 @@
 	$(INSTALL_DATA) glib-enum-types.h $(RUBYARCHDIR)
 
 install:
-	$(INSTALL_DATA) ../glib-2.0-ruby.pc $(libdir)/pkgconfig
+	$(INSTALL_DATA) ../glib-2.0-ruby.pc -D $(libdir)/pkgconfig/glib-2.0-ruby.pc
--- glib/src/depend.ori	2009-09-03 19:00:07.000000000 +0700
+++ glib/src/depend	2009-09-03 19:00:37.000000000 +0700
@@ -6,4 +6,4 @@
 	$(INSTALL_DATA) glib-enum-types.h $(RUBYARCHDIR)
 
 install:
-	$(INSTALL_DATA) ../glib-2.0-ruby.pc $(libdir)/pkgconfig
+	$(INSTALL_DATA) ../glib-2.0-ruby.pc -D $(libdir)/pkgconfig/glib-2.0-ruby.pc
